share/functions/help.fish

```diff
@@ -78,6 +78,11 @@ function help --description 'Show help for the fish shell'
 		return 1
 	end
 
+        # In Cygwin, start the user-specified browser using cygstart
+        if begin; type -q cygstart; and test $fish_browser != "cygstart"; end
```

Comment:
We don't currently have backports, so the cygwin people will have to pick this one up themselves.